### DISCUSSION TITLE
Codespaces iPad: Cannot Repeat Keys

### DISCUSSION THREAD

**Discussion by michaelperel** [community/community#40084] [Codespaces]
### Select Topic Area

Bug

### Body

When using codespaces on the iPad, you cannot hold down a key to repeat it (Safari + Magic Keyboard + iPad Pro + iOS 15.7.1). For instance, if you press and hold “i”, it will only appear once. For people that use the vim extension, this means that you cannot scroll by holding down a key (which severely limits the usability of vim).

This problem used to exist in various iOS terminal apps such as blink and termius but has been fixed.

**Comment by ossanna16** [comment]
Hi there @michaelperel and welcome to our community! Thank you for reporting this!

**Comment by michaelnoisternig-tomtom** [comment]
It's not just there, this idiotic "feature" is everywhere. Not even in PR comments repeat keys (i.e. holding down a key to continue it being evoked) work. However, curiously, it works here in discussion comments.
